person.rb

```diff
@@ -26,4 +27,9 @@ def correct_name
   def of_age?
     @age >= 18
   end
+
+  def add_rentals(rentals)
+    @rentals.push(rentals)
+    book.rentals = self
+  end
```

Comment:
- [ ] Same here, This method supposes to add a new rental and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association, please fix that.